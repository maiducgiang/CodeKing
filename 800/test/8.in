10
63
19 24 52 8 9 45 57 45 28 40 45 23 7 4 14 32 40 26 44 39 25 46 3 29 4 30 61 46 20 32 35 5 32 43 49 20 31 26 32 23 46 60 56 24 25 53 37 43 3 19 33 55 9 6 28 3 43 55 52 33 48 25 5 
56
23 54 42 49 21 11 23 56 24 15 21 56 30 32 45 41 2 1 38 18 12 43 17 40 37 1 41 21 48 5 56 51 40 6 17 5 53 5 27 3 11 32 6 53 30 30 5 33 49 39 20 2 14 22 30 43 
65
3 35 20 2 40 51 24 50 34 40 11 4 27 22 12 27 20 47 20 20 60 1 16 58 7 7 23 16 14 48 38 34 50 14 20 16 34 34 9 36 11 30 19 61 60 11 53 60 13 7 60 4 29 38 18 58 47 38 65 7 38 5 28 6 45 
44
4 20 40 43 32 4 31 34 15 12 30 38 9 16 32 1 25 3 33 14 6 14 10 9 12 4 32 29 18 22 37 8 37 31 5 29 12 3 19 13 44 14 43 25 
61
10 46 35 13 42 15 6 49 7 57 49 61 23 44 60 31 46 39 43 59 10 44 42 17 18 34 17 40 45 59 15 46 15 16 56 51 12 21 50 1 48 24 21 2 38 30 36 28 40 41 15 55 4 7 19 6 14 56 15 36 14 
41
21 12 32 13 40 38 1 34 2 5 28 10 18 2 23 6 26 21 31 32 30 4 37 5 25 35 8 7 34 36 2 25 19 24 40 30 34 5 3 16 38 
65
61 58 20 11 55 35 59 18 22 64 35 4 4 26 35 18 1 55 24 34 2 3 58 39 48 26 63 52 32 36 49 12 56 40 22 39 52 12 51 50 15 3 22 12 58 39 43 57 2 38 32 11 35 55 26 25 32 19 61 26 28 32 44 36 33 
45
4 28 25 20 24 35 6 31 33 33 13 40 22 18 8 33 41 18 43 30 16 3 2 41 3 44 37 34 18 10 42 17 14 28 22 15 29 17 21 18 15 41 40 8 45 
69
49 46 47 40 45 45 51 56 17 8 55 7 3 68 66 55 44 28 58 45 57 49 65 12 48 64 20 6 31 47 1 49 38 16 35 64 54 27 43 29 10 38 55 41 50 30 67 20 4 52 63 52 48 19 4 11 51 4 63 9 49 1 68 24 50 33 39 28 5 
48
18 48 40 16 12 31 45 26 40 28 29 15 34 43 47 11 24 32 16 17 34 36 32 6 6 45 18 3 43 19 10 1 2 42 26 29 20 26 31 11 44 19 14 6 22 18 29 31 
