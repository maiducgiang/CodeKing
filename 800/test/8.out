1
31
29
20
34
12
17
18
25
19
